src/helpers/error-message.helper.ts

```diff
@@ -138,7 +143,9 @@ export class ErrorMessageHelper {
         title = `${error.worker}`;
         break;
       case ParseErrorCode.UNKNOWN_VALUE:
-        message = `Nieznana wartość zmiany: "${error.actual}" w dniu ${error.day! + 1}.`;
+        message = `Nieznana wartość zmiany: "${error.actual}" w dniu ${
+          error.day! + 1
+        }. Obecnie pole jest puste. Możesz ręcznie przypisać zmianę z tych już istniejących lub przedtem utworzyć nową.`;
```

Comment:
_przedtem utworzyć nową_ dla mnie to jest lekko dziwacznie sformułowane, ale jak jesteście pewni, że jest git, to nie będę się czepiał.